src/script/properties/PropertiesRepository.js

```diff
@@ -208,10 +208,16 @@ class PropertiesRepository {
 
   // Map a property and set it into our state
   setProperty(key, value) {
-    this.logger.info(`Setting key "${key}" to value "${value}"...`);
+    this.logger.info(`Setting key "${key}"...`, value);
+
     switch (key) {
+      case PropertiesRepository.CONFIG.WEBAPP_ACCOUNT_SETTINGS:
+        if (this.properties.version === value.version) {
+          this.properties = {...this.properties, ...value};
```

Comment:
Object spread is not supported in the webapp. You need to use `Object.assign` here